### PR TITLE
add links to dbw docs

### PR DESCRIPTION
**Pull request by kaycebasques**
@ebidel plz take a looksie

### CODE REVIEW

File: lighthouse-core/audits/dobetterweb/external-anchors-use-rel-noopener.js [hunk 1]
```diff
@@ -30,7 +30,9 @@ class ExternalAnchorsUseRelNoopenerAudit extends Audit {
       category: 'Performance',
       name: 'external-anchors-use-rel-noopener',
       description: 'Site opens external anchors using rel="noopener".',
-      helpText: 'Anchors that open in a new tab should use <code>target="_blank"</code> and <code>rel="noopener"</code> so the <a href="https://jakearchibald.com/2016/performance-benefits-of-rel-noopener" target="_blank">opening page\'s performance does not suffer.</a>',
+      helpText: 'Open new tabs using <code>rel="noopener"</code> to improve performance and ' +
+                'prevent security vulnerabilities. <a href="https://developers.google.com/web/tools/lighthouse/audits/noopener" ' +
+                'target="_blank" rel="noopener noreferrer">Learn more</a>.',
```
Comment (ebidel) [inline lighthouse-core/audits/dobetterweb/external-anchors-use-rel-noopener.js]: oh boom. `rel="noopener noreferrer"`. Practicing what we preach.

Do we want to keep the referrer though? When we have the online viewer, there will actually be a url referral.

File: lighthouse-core/audits/dobetterweb/no-console-time.js [hunk 2]
```diff
@@ -35,7 +35,11 @@ class NoConsoleTimeAudit extends Audit {
       category: 'JavaScript',
       name: 'no-console-time',
       description: 'Site does not use console.time() in its own scripts',
-      helpText: 'Consider using the <a href="https://developer.mozilla.org/en-US/docs/Web/API/User_Timing_API" target="_blank">User Timing API</a> (<a href="https://developer.mozilla.org/en-US/docs/Web/API/Performance/mark" target="_blank">performance.mark()</a> and <a href="https://developer.mozilla.org/en-US/docs/Web/API/Performance/measure" target="_blank">performance.measure()</a>), which is a standard, uses high resolution timestamps, and has the added benefit of integrating with the DevTools timeline.',
+      helpText: 'Consider using the User Timing API instead. ' +
```
Comment (ebidel) [inline lighthouse-core/audits/dobetterweb/no-console-time.js]: This should call out the APIs like the no-datenow audit

Consider using <code>performance.mark()</code> and <code>performance.measure()</code> from the...

File: lighthouse-core/audits/dobetterweb/uses-http2.js [hunk 3]
```diff
@@ -36,7 +36,10 @@ class UsesHTTP2Audit extends Audit {
       category: 'Performance',
       name: 'uses-http2',
       description: 'Site uses HTTP/2 for its own resources',
-      helpText: 'h2 offers many benefits over its http/1.1 predecessor: binary, multiplexing, server push, etc. See <a href="https://http2.github.io/faq/" target="_blank">this FAQ</a> for more information.',
+      helpText: 'HTTP/2 offers many benefits over HTTP/1.1, including binary data ' +
```
Comment (ebidel) [inline lighthouse-core/audits/dobetterweb/uses-http2.js]: including binary data -> including binary headers

File: lighthouse-core/audits/dobetterweb/uses-http2.js [hunk 4]
```diff
@@ -36,7 +36,10 @@ class UsesHTTP2Audit extends Audit {
...
+                'transfers, multiplexing, and server pushes. ' +
```
Comment (ebidel) [inline lighthouse-core/audits/dobetterweb/uses-http2.js]: pushes -> push

File: lighthouse-core/audits/dobetterweb/uses-passive-event-listeners.js [hunk 5]
```diff
@@ -42,7 +42,10 @@ class PassiveEventsAudit extends Audit {
       category: 'JavaScript',
       name: 'uses-passive-event-listeners',
       description: 'Site uses passive listeners to improve scrolling performance',
-      helpText: `<a href="https://www.chromestatus.com/features/5745543795965952" target="_blank">Passive event listeners</a> enable better scrolling performance. If you don't call <code>preventDefault()</code> in your <code>${this.SCROLL_BLOCKING_EVENTS.toString()}</code> event listeners, make them passive: <code>addEventListener('touchstart', ..., {passive: true})</code>.`,
```
Comment (ebidel) [inline lighthouse-core/audits/dobetterweb/uses-passive-event-listeners.js]: while you're in this file, would you also mind updating the the code comments that say this audit only looks at document-level listeners. That is now longer true. It looks at all listeners on the page.

https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/audits/dobetterweb/uses-passive-event-listeners.js#L20

https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/audits/dobetterweb/uses-passive-event-listeners.js#L62

File: lighthouse-core/audits/dobetterweb/external-anchors-use-rel-noopener.js (hunk 1 above)
Comment (kaycebasques) [inline lighthouse-core/audits/dobetterweb/external-anchors-use-rel-noopener.js]: I've been using `noreferrer` in the links b/c the [target="_blank" considered harmful](https://mathiasbynens.github.io/rel-noopener/) post suggested using it to safeguard older browsers. LMK how you want to proceed.

Comment (kaycebasques) [inline lighthouse-core/audits/dobetterweb/external-anchors-use-rel-noopener.js]: But on second thought, I'd like to collect referrer stats for the /web GA account, so I think I will remove it

File: lighthouse-core/audits/dobetterweb/no-console-time.js (hunk 2 above)
Comment (kaycebasques) [inline lighthouse-core/audits/dobetterweb/no-console-time.js]: ✔️

File: lighthouse-core/audits/dobetterweb/external-anchors-use-rel-noopener.js (hunk 1 above)
Comment (kaycebasques) [inline lighthouse-core/audits/dobetterweb/external-anchors-use-rel-noopener.js]: ✔️

File: lighthouse-core/audits/dobetterweb/uses-http2.js (hunk 4 above)
Comment (kaycebasques) [inline lighthouse-core/audits/dobetterweb/uses-http2.js]: ✔️

File: lighthouse-core/audits/dobetterweb/uses-passive-event-listeners.js (hunk 5 above)
Comment (kaycebasques) [inline lighthouse-core/audits/dobetterweb/uses-passive-event-listeners.js]: ✔️

File: lighthouse-core/audits/dobetterweb/external-anchors-use-rel-noopener.js (hunk 1 above)
Comment (ebidel) [inline lighthouse-core/audits/dobetterweb/external-anchors-use-rel-noopener.js]: Yea. It's also us redirecting to us :)

**Comment by kaycebasques** [comment]
@ebidel I updated the thingies!

**Comment by ebidel** [comment]
SOLID

**Comment by ebidel** [comment]
@kaycebasques can you fix the failing travis tests. Some of these have line lengths that are too long.

**Comment by kaycebasques** [comment]
I fixed the thingies

File: lighthouse-core/audits/dobetterweb/external-anchors-use-rel-noopener.js [hunk 6]
```diff
@@ -29,8 +29,10 @@ class ExternalAnchorsUseRelNoopenerAudit extends Audit {
     return {
       category: 'Performance',
       name: 'external-anchors-use-rel-noopener',
-      description: 'Site opens external anchors using rel="noopener".',
-      helpText: 'Anchors that open in a new tab should use <code>target="_blank"</code> and <code>rel="noopener"</code> so the <a href="https://jakearchibald.com/2016/performance-benefits-of-rel-noopener" target="_blank">opening page\'s performance does not suffer.</a>',
+      description: 'Site opens external anchors using rel="noopener"',
+      helpText: 'Open new tabs using <code>rel="noopener"</code> to improve performance and ' +
```
Comment (brendankenny) [inline lighthouse-core/audits/dobetterweb/external-anchors-use-rel-noopener.js]: would it be terrible to turns these from horizontal alignment after the line break to just four space indentation to match the usual line break style we use?

Comment (paulirish) [inline lighthouse-core/audits/dobetterweb/external-anchors-use-rel-noopener.js]: If we're bikeshedding the string formatting... what about using multiline strings? Seems easier to maintain.

```js
{
  category: 'Performance',
  name: 'external-anchors-use-rel-noopener',
  description: 'Site opens external anchors using rel="noopener"',
  helpText: `Open new tabs using <code>rel="noopener"</code> to improve performance and 
		prevent security vulnerabilities. <a href="https://developers.google.com/web/
        tools/lighthouse/audits/noopener" target="_blank" rel="noopener">Learn more</a>.`,
  requiredArtifacts: ['URL', 'AnchorsWithNoRelNoopener']
}
```

Comment (kaycebasques) [inline lighthouse-core/audits/dobetterweb/external-anchors-use-rel-noopener.js]: Your guys' call but LMK in this thread how to proceed

Comment (ebidel) [inline lighthouse-core/audits/dobetterweb/external-anchors-use-rel-noopener.js]: Personally, it's not a big deal for me.

Comment (kaycebasques) [inline lighthouse-core/audits/dobetterweb/external-anchors-use-rel-noopener.js]: @paulirish is there a way to keep the multiline from donking up my URLs?

When I format like this:

screen shot 2016-12-16 at 1 37 26 pm


It adds space to my URL (see the hover at bottom of Chrome window):

screen shot 2016-12-16 at 1 35 56 pm

Comment (ebidel) [inline lighthouse-core/audits/dobetterweb/external-anchors-use-rel-noopener.js]: Yea, this is the problem with template strings. It'll be inclusive of whitespace. You could break up links and concat those parts`+ ... +`, but I'm not sure it's worth the trouble to mix and match template strings and str concat.

**Comment by paulirish** [comment]
Ya just can't newline within an attribute value.

File: lighthouse-core/audits/dobetterweb/external-anchors-use-rel-noopener.js (hunk 6 above)
Comment (brendankenny) [inline lighthouse-core/audits/dobetterweb/external-anchors-use-rel-noopener.js]: > Not sure if our linter will complain though....

we have `ignoreUrls` turned on for our max line length check, so any line with a URL in it won't fail the lint check.

**Comment by brendankenny** [comment]
I personally think without `\n`s in the string there's no intention to have line breaks, so the ability for template literals to have new lines in them that are stripped out when used in html is a little distracting.

That said, it doesn't really matter to me if it's four space indentation or template literals or one really long line :) We should pick one, though, as I'd like to get this into the release today.

**Comment by kaycebasques** [comment]
How about:

* Update to use four-line indents.
* Revisit literals later.

I can update for release today if we do ^

**Comment by brendankenny** [comment]
SGTM :)

**Comment by kaycebasques** [comment]
get me out of this god-forsaken bikeshed

(updated & rebased)

**Comment by ebidel** [comment]
Yay! Let's get this in for the release.

**Pull request merged**